index.html

```diff
@@ -0,0 +1,124 @@
+        </a>
+        <br>
+        <a class="white flex" href="#">
+          <h3 class="h1">
+            Watch the "Your Verse" film and explore the stories
+            &ensp;
+            <i class="fa fa-play-circle" aria-hidden="true"></i>
+          </h3>
+        </a>
+      </div>
+    </section>
+    <article class="gallery flex">
+      <a href="#"><img src="images/promo_iphone5c.jpg" alt="gallery image iPhone"></a>
+      <a href="#"><img src="images/promo_iphone5s.jpg" alt="gallery image iPhone"></a>
+      <a href="#"><img src="images/promo_30_years.jpg" alt="gallery image years"></a>
+      <a href="#"><img src="images/promo_supplier_responsibility.jpg" alt="supplier image"></a>
+    </article>
+  </section>
+  <footer class="main">
+    <div class="section-one">
+      <p>Shop the
+        <a href="https://web.archive.org/web/20140301004610/http://store.apple.com/us">Apple Online Store</a>
+        (1-800-MY-APPLE), visit an <a
+          href="https://web.archive.org/web/20140301004610/http://www.apple.com/retail/">Apple Retail Store</a> , or
+        find <a href="https://web.archive.org/web/20140301004610/http://www.apple.com/buy/">reseller</a></p>
+        <div class="section-three">
+           <p> <small>Copyright © 2014 Apple Inc. All rights reserved.<a class="terms" href="https://web.archive.org/web/20140301004610/http://www.apple.com/legal/">Terms of Use</a><a href="https://web.archive.org/web/20140301004610/http://www.apple.com/legal/privacy/">Privacy Policy</a>
+          </small>
+        </p>
+        </div>
+      
+    </div>
+    <div class="section-two">
+      <ul class="flex">
+        <li>
+          <a href="https://web.archive.org/web/20140301004610/http://www.apple.com/sitemap/">Site Map</a>
+        </li>
+        <li>
+          <a href="https://web.archive.org/web/20140301004610/http://www.apple.com/hotnews/">Hot News</a>
+        </li>
+        <li>
+          <a href="https://web.archive.org/web/20140301004610/http://www.apple.com/rss/">RSS Feeds</a>
+        </li>
+        <li>
+          <a href="https://web.archive.org/web/20140"></a>
+        </li>
+        <li>
+          <a href="https://web.archive.org/web/20140301004610/http://www.apple.com/pr/">Media Info</a>
+        </li>
+        <li>
+          <a href="https://web.archive.org/web/20140301004610/http://www.apple.com/environment/">Environment</a>
+        </li>
+        <li>
+          <a href="https://web.archive.org/web/20140301004610/http://www.apple.com/jobs/">Job Opportunities</a>
+        </li>
+        <li>
+          <a href="https://web.archive.org/web/20140301004610/http://www.apple.com/contact/">Contact Us</a>
+        </li>
+        <li>
+          <a href="https://web.archive.org/web/20140301004610/http://www.apple.com/choose-your-country/">
+            <img src="images/usa.png" alt="">
+          </a>
+        </li>
+      </ul>
+    </div>
+    
+  </footer>
```

Comment:
- [ ] As was pointed out to you in a previous review, this section has too much extra space above it. Please reduce the space between the footer and the section above it by adjusting the margins so it looks more like the original page. 


| Source Page	| Clone Page |
|:---:|:---:|
|![Screen Shot 2020-07-30 at 6 43 59 PM](https://user-images.githubusercontent.com/48804258/88950224-a268ee80-d294-11ea-90dd-a1fdfb4e6cc7.png) |![Screen Shot 2020-07-30 at 6 43 37 PM](https://user-images.githubusercontent.com/48804258/88950196-95e49600-d294-11ea-93cd-7a03264dbd81.png)|

Reply:
As you can see the footer now is set there is no space